docs/reference/query-dsl/function-score-query.asciidoc

```diff
@@ -153,8 +153,16 @@ GET /_search
 --------------------------------------------------
 // TEST[setup:twitter]
 
-NOTE: Scores produced by the `script_score` function must be non-negative,
-otherwise an error will be thrown.
+[IMPORTANT]
+====
+{es} stores scores produced by the `script_score` function as positive
+32-bit floating point numbers in the `_score` meta-field.
+
+As a result, scores are limited to 32-bits of precision. Larger numbers will
+lose precision during storage.
```

Comment:
I waiver over whether these sentences are needed. Any feedback appreciated!

Reply:
Maybe we should clarify that _all_ scores are 32-bit floats in Elasticsearch, this is not specific to this query. Then if a script_score produces a value that has greater accuracy than a 32-bit float, it will be automatically converted to the nearest 32-bit float.

Unrelated note: I find the word "storage" possibly confusing since we don't store scores on disk, which is what some users might think this statement implies.

Reply:
Thanks @jpountz. Very good point re: store/storage.

I reworded this section as you suggested with 63f714ac64960daf58c03b2c3548178a42d90f91. I also noted that `_score` is 32-bit in the Search API reference docs.